### PR TITLE
[doc] Remove references to obsolete BuildApplet on macOS.

### PR DESCRIPTION
**Pull request by andresdelfino**

### CODE REVIEW

**Comment by tirkarthi** [comment]
cc: @ned-deily

File: Doc/using/mac.rst [hunk 1]
```diff
@@ -159,7 +159,7 @@ https://riverbankcomputing.com/software/pyqt/intro.
 Distributing Python Applications on the Mac
 ===========================================
 
-The "Build Applet" tool that is placed in the MacPython 3.6 folder is fine for
```
Comment (ned-deily) [inline Doc/using/mac.rst:162]: Thanks for catching this.  The Build Applet tool does not exist anymore.  So this whole paragraph should simply be deleted.

**Review by ned-deily — CHANGES_REQUESTED** [review]

**Comment by andresdelfino** [comment]
I didn't expect the Spanish Inquisition!

**Review by ned-deily — APPROVED** [review]
Thanks for the update and catching the other reference.

**Pull request merged**